server/bin/immich-healthcheck

```diff
@@ -1,3 +1,23 @@
 #!/usr/bin/env bash
 
-node /usr/src/app/dist/bin/healthcheck.js
+if [[ "$IMMICH_WORKERS_EXCLUDE" == *"api"* ]] ||
+   [[ "$IMMICH_WORKERS_INCLUDE" != *"api"* ]]; then
```

Comment:
Has this been tested? AFAIK by default all workers will run, so it doesn't make sense to test for api in the include variable since most people will not have this set.

Should probably be something like `if [[ ( $IMMICH_WORKERS_INCLUDE != '' && $IMMICH_WORKERS_INCLUDE != *api* ) || $IMMICH_WORKERS_EXCLUDE == *api* ]]`

https://immich.app/docs/install/environment-variables

Reply:
Yes, I've tested it, but you're right. I wrote the include test to handle ``IMMICH_WORKERS_INCLUDE=microservices``, but it does make more sense to just test the excludes